Models/ToyLanding/ToyLandingModel.osim

```diff
@@ -8228,7 +8228,7 @@
 						<objects>
 							<HuntCrossleyForce::ContactParameters>
 								<!--Names of geometry objects affected by these parameters.-->
-								<geometry>platform heel_r ball_big_toe_r small_toe_r</geometry>
+								<geometry>platformSurface heel_r ball_big_toe_r small_toe_r</geometry>
```

Comment:
Perhaps `platform_surface` would be more in keeping with the naming convention in this model.

Reply:
Sure. :pig: :shit: :lipstick: